### ISSUE TITLE
design new About Us page

### ISSUE THREAD

**Issue by anitadesigns** [hackforla/expunge-assist#1050]
### Overview
The About Us page is not consistent with the other main pages of the websites - it is missing the header graphic that the other pages have. Content is also completing a redesign of the copy on the page and the alignment will need to be adjusted once that is complete.

### Action Items
- [x] Re-visit the About Us page on Figma and go over the progress 
- [x] Brainstorm ideas with Content
- [x] Develop a new layout for the About Us page and add illustrations that would complement the content on the page (Desktop and mobile frames needed)
- [x] Show Content 
- [x] Present to Dev 
- [x] Present to Design Team for Feedback 
- [x] Update wireframes on Master pages in Figma (Desktop and mobile)
- [x] hand off to dev: #1384 
   - [x] add design
   - [x] remove `dependency` label
   - [x] move to `new issue approval` column
   - [x] add `ready for dev lead` label

### Resources/Instructions
- Link to [Figma Workspace](https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?type=design&node-id=2-45&mode=design&t=Lv5OeR7xsBXFGNMH-0) for this issue
- Previously linked to linked to #684 and #965
- MAIN: #1365

**Comment by anitadesigns** [comment]
Content is ready to work on this issue together. @SamHyler and @emmathrash are leading this. Design is seeking a designer to assign this issue to

**Comment by Ugochukwu-On** [comment]
Steady progress, content team are deciding the best structure for the page's design.

**Comment by Ugochukwu-On** [comment]
> Steady progress, content team are deciding the best structure for the page's design. 

It should be done in 2 weeks

**Comment by anitadesigns** [comment]
Onwubu worked on this issue for a bit and reformatted the layout. @jyehllow and I will review it to ensure it is consistent with the rest of the website's design. We will then hand it over to Dev. 

Content shared that they made quick fixes to the copy for now and then will return to it at a later time

**Comment by vanessaavviles** [comment]
Please provide update:

Progress:
Blockers:
Availability:
ETA:

**Comment by anitadesigns** [comment]
**Progress:** Received an update from Content. The copy on the page may need more major changes than we planned to tackle for this iteration. Waiting to confirm the gravity of this issue. If we decide to continue with major changes then we will need to re-design the page. 

**Blockers:** Currently discussing next steps with Content. If we decide to redesign then this issue will need to be unassigned and then moved to prioritized backlog and add a dependency to the Content specific issue. 

**ETA:** Unkown at this time

**Comment by anitadesigns** [comment]
Progress: Content is finalizing the copy and we are starting to add the illustrations to each section

Blockers: Trying to find open source illustrations that are not bias for the "Record Clearance" section 

ETA: next week

**Comment by anitadesigns** [comment]
Progress: Added illustrations and now sharing with @TechWriterMelissa to ensure illustrations match voice and tone before handing over

Blockers: none

ETA: end of this week
